src/auth/jwt.rs

```diff
@@ -67,11 +67,9 @@ pub fn decode_token(token: &str) -> anyhow::Result<Claims> {
     let jwt_secret = JWT_SECRET
         .read()
         .map_err(|e| AuthError::ReadJwtSecret(e.to_string()))?;
-    let token_data = decode::<Claims>(
-        token,
-        &DecodingKey::from_secret(&jwt_secret),
-        &Validation::default(),
-    )?;
+    let mut validation = Validation::default();
+    validation.validate_exp = false; // Disable expiration validation
```

Comment:
No need to make `validation` mutable.

```rust
let validation = Validation {
    validate_exp: false,
    ..Validation::default()
};
```

Reply:
Since `Validation::validate_signature` is scoped as `pub(crate)`, struct literal initialization with `..Validation::default()` isn’t supported. Therefore, I’ll keep the current implementation in the PR as-is.